api/agent/stats.go

```diff
@@ -200,3 +157,61 @@ func (s *stats) Stats() Stats {
 	s.mu.Unlock()
 	return stats
 }
+
+// Constants used when constructing span names and field keys for these metrics
+var queuedSuffix = "queued"
+var callsSuffix = "calls"
+var runningSuffix = "running"
+var completedSuffix = "completed"
+var failedSuffix = "failed"
```

Comment:
go thing: these look like consts more than vars:

```go
const (
  queuedSuffix = "queued"
  // ...
)
```

if passed as args (as I see, for these) that take string parameters, they will get turned into strings. constants in go are pretty cool under the hood, this blog post is pretty interesting https://blog.golang.org/constants